### ISSUE TITLE
XPU Trace fails on PReLU size

### ISSUE THREAD

**Issue by xsacha** [intel/intel-extension-for-pytorch#265]
Incorrect error on input size for PReLU. Works on all other backends except for XPU.

Reproducer:
```Python
import torch
import torch.nn as nn
import intel_extension_for_pytorch as ipex

if __name__ == '__main__':
    #x = nn.Sequential(nn.Conv2d(3, 64, (3,3), 1, 1), nn.BatchNorm2d(64))
    x = nn.Sequential(nn.Conv2d(3, 64, (3,3), 1, 1), nn.BatchNorm2d(64), nn.PReLU(64))
    inp = torch.randn(1,3,112,112)
    traced = torch.jit.trace(x, inp)
    traced.to('xpu')
    inp = inp.to('xpu')
    # Run twice
    out = traced(inp)
    out = traced(inp)
```

If the input size is changed to 64, it works. If the PReLU is removed, it works.

**Comment by gujinghui** [comment]
Thanks for the post.
This is caused by double parameter in BatchNorm2d, while double is not supported on your HW.
We will try to fix it in next release.

**Comment by xsacha** [comment]
Is there a way to make the BatchNorm2d use float instead of double then?

**Comment by gujinghui** [comment]
As a temp workaround, you can try to static_cast the momentum to float in below lines.
We will fix it in next release.

https://github.com/intel/intel-extension-for-pytorch/blob/9b770aaab9bbf2fe6e21f129387c83e425784111/csrc/aten/operators/BatchNorm.cpp#L52

https://github.com/intel/intel-extension-for-pytorch/blob/9b770aaab9bbf2fe6e21f129387c83e425784111/csrc/aten/operators/BatchNorm.cpp#L68
